src/player.h

```diff
@@ -1452,6 +1452,8 @@ class player : public Character
         int oxygen;
         int stamina;
         double recoil = MAX_RECOIL;
+        std::weak_ptr<Creature> last_target;
+        tripoint last_target_pos;
```

Comment:
Please wrap this into `cata::optional` instead of using an undocumented sentinel value.